addons/binding/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/BindingServlet.java

```diff
@@ -99,8 +99,9 @@ protected void doGet(@Nullable HttpServletRequest req, @Nullable HttpServletResp
         }
 
         StringBuilder html = new StringBuilder();
-        html.append("<html><head><title>" + StringEscapeUtils.escapeHtml(BINDING_NAME) + "</title><head><body>");
-        html.append("<h1>" + StringEscapeUtils.escapeHtml(BINDING_NAME) + "</h1>");
+        html.append(
+                "<html><head><title>" + StringEscapeUtils.escapeHtml(BINDING_NAME + " RC3") + "</title><head><body>");
```

Comment:
What is this about? There isn't any RC3...

Reply:
Ah, ok, I just saw that you did that for the intermediate release to the marketplace. All fine.